integration/README.md

```diff
@@ -3,45 +3,70 @@
 
 Integration tests for Cook scheduler
 
-To run the tests on a local cook install, run
+## Running in docker
+
+If you're running cook in docker, it can be beneficial to run the integration tests in a docker container so that the IP addresses resolve correctly. To do so, there are two separate scripts:
 
 ```bash
-$ python setup.py nosetests
+$ bin/build-docker-image.sh
 ```
 
-To configure the Cook scheduler URL, set the `COOK_SCHEDULER_URL` environment variable. The default value is `http://localhost:12321`.
+This needs to be run once, and can be run again in order to include any new dependencies in the docker image.
 
-If you want to run a single test and see the log messages as they occur, you can run, for example:
+```bash
+$ bin/run-integration.sh COOK_PORT_1 COOK_PORT_2
+```
+
+This launches the integration tests in the docker container. You can optionally pass one (or two, if using `COOK_MULTI_CLUSTER`) ports that cook is listening on. This mounts the local directory in the docker container so there is no need to rebuild when changing the tests.
+
+## Running the tests locally
+
+Running the integration tests from your local machine may simplify your development workflow.
+
+### Installing dependencies
+
+Our integration tests currently require Python 3.6.
+All dependencies are specified in [`requirements_dev.txt`](./requirements_dev.txt),
```

Comment:
It looks like this file is being deleted

Reply:
It was renamed. I thought I fixed this, but I might have lost it in a rebase. Good catch! I'll do a grep to make sure I didn't miss it anywhere else.